src-electron/generator/helper-endpointconfig.js

```diff
@@ -645,22 +645,23 @@ function asMEI(manufacturerCode, code) {
 
 // The representation of null depends on the type, so we can't use a single
 // macro that's defined elsewhere for "null value".
-function determineAttributeDefaultValue(
+async function determineAttributeDefaultValue(
   specifiedDefault,
   type,
   typeSize,
-  isNullable
+  isNullable,
+  db,
+  sessionId
 ) {
-  if (specifiedDefault !== null || !isNullable) {
+  if (specifiedDefault && (specifiedDefault !== null || !isNullable)) {
     return specifiedDefault
   }
 
   if (types.isString(type)) {
     // Handled elsewhere.
     return null
   }
-
-  if (types.isSignedInteger(type)) {
+  if (await types.isSignedInteger(db, sessionId, type)) {
```

Comment:
This signature has not been changed in `validation.js -> types.isSignedInteger(attribute.type)`

Reply:
ohh I see
thank you